karrot/groups/api.py

```diff
@@ -201,6 +201,23 @@ def trust_user(self, request, pk, user_id):
 
         return Response(data={})
 
+    @trust_user.mapping.delete
```

Comment:
Ah, nice find using this [`mapping` thing](https://www.django-rest-framework.org/api-guide/viewsets/#routing-additional-http-methods-for-extra-actions), we could clean up other actions using that pattern one day! :+1: